CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [0.19.4] - 2025-06-17
+
+### Fixed
+
+- move manifest validation to another place in the preparation to safeguard
+  against potential future security issues.
```

Comment:
I would mention this as "security hardening".

Reply:
I'd probably put "move manifest validation later in the preparation to harden against possible cargo changes.". Mentioning a potential future issue implies we are aware there is going to be an issue in the future.

Reply:
updated